src/pages/AddressBookContents.tsx

```diff
@@ -92,7 +127,7 @@ export default function AddressBookContents(props) {
           if (!contactInfo) continue;
```

Comment:
Please check if contactInfo is defined

Reply:
This is require because if contactInfo comes null then it start the process again via continue.